nixos/modules/services/system/dbus.nix

```diff
@@ -51,6 +54,12 @@ in
         '';
       };
 
+      apparmor = mkOption {
+        type = types.enum [ true false "required" ];
```

Comment:
Those selections don't make the clearest sense to me, like two booleans and a string? I'd be better if they were all the same type, can we just make this `enabled` `disabled` `required` directly and then use those values directly? I think it's simpler and feels clearer anyways.

Reply:
Good point, I'll rebase this later